[0.6.0-UT] unskip selected test cases

phambinhfin opened the pull request:
Unskipped the following tests:
- lax_control_flow_test: testAssociativeScanSolvingRegressionTest
- scipy_signal_test: testWelchWithDefaultStepArgsAgainst
- sparse_test: cuSparseTest, SparseObjectTest
- random_lax_test: DistributionsTest::testOrthogonal{1,7,8,9}
- pallas_test: PallasCallInterpretTest::test_matmul (various shapes/dtypes)
- lax_numpy_test: testPolyMul1
- gpu_ops_test: FusedAttentionInterpretTest::test_fused_attention_bwd{4,6,7,8} (change the sample test data)
- export_harnesses_multi_platform_test: PrimitiveTest::test_prim_qr_multi_array_shape (various)
- ann_test: pass for whole file

gulsumgudukbay reviewed (changes requested):
Could you please let us know in detail how you tested these? The ROCm build number/container/testing process (if you used the CI scripts).

Arech8 commented on tests/pallas/gpu_ops_test.py:
```diff
@@ -235,12 +235,12 @@ def impl(q, k, v):
-              ("block_kv_dkv", 16),
-              ("block_q_dq", 16),
-              ("block_kv_dq", 64),
+              ("block_q", 128),
```
I think this change also affects NVDA, right? Why does it have to be changed?

gulsumgudukbay replied:
No, note the `if jtu.is_device_rocm()` statement in line 254

phambinhfin committed:
unskip selected test cases

    Unskipped the following tests:
    - lax_control_flow_test: testAssociativeScanSolvingRegressionTest
    - scipy_signal_test: testWelchWithDefaultStepArgsAgainst
    - sparse_test: cuSparseTest, SparseObjectTest
    - random_lax_test: DistributionsTest::testOrthogonal{1,7,8,9}
    - pallas_test: PallasCallInterpretTest::test_matmul (various shapes/dtypes)
    - lax_numpy_test: testPolyMul1
    - gpu_ops_test: FusedAttentionInterpretTest::test_fused_attention_bwd{4,6,7,8} (change the sample test data)
    - export_harnesses_multi_platform_test: PrimitiveTest::test_prim_qr_multi_array_shape (various)
    - ann_test: pass for whole file

phambinhfin commented:
@gulsumgudukbay I follow development setup, using a fresh ubuntu 22.04 and install necessary stuffs inside using `bash tools/docker_dev_setup.sh` . Then also build jaxlib manually because the fix https://github.com/ROCm/jax/pull/525

phambinhfin commented:
@Ruturaj4  I clear the comment
